src/connectWithoutActions.js

```diff
@@ -0,0 +1,8 @@
+/* @noflow */
+
+import { connect } from 'react-redux';
+
+import type { MapStateToProps } from './types';
+
+export default (mapStateToProps: MapStateToProps) => (component: React$Component<*, *, *>) =>
```

Comment:
Not sure if this file is really needed or directly can be used!

Reply:
The use of this can be replaced with just calling `connect` directly, right? That seems better.